.gitpod.yml

```diff
@@ -0,0 +1,53 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - before: |
+      export GHCUP_INSTALL_BASE_PREFIX=/workspace
+      export BOOTSTRAP_HASKELL_NONINTERACTIVE=1
+      export BOOTSTRAP_HASKELL_MINIMAL=1
+      export CABAL_DIR=/workspace/.cabal
+      export STACK_ROOT=/workspace/.stack
+      curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
+
+      # ensure that there is an HLS binary for the installed GHC version
+      /workspace/.ghcup/bin/ghcup install ghc 8.10.5 --set
```

Comment:
Is there a specific reason for using 8.10.5 instead of any later version of GHC?

Reply:
Yes, is the comment not helpful?

Reply:
Ah, I misunderstood the comment. I see it now, sorry for bothering you.